integration_tests/test_dbapi.py

```diff
@@ -68,6 +68,20 @@ def test_select_query_result_iteration(presto_connection):
     assert len(list(rows0)) == len(rows1)
 
 
+def test_select_cursor_iteration(presto_connection):
+    cur0 = presto_connection.cursor()
+    cur0.execute("select custkey from tpch.sf1.customer LIMIT 10")
+    rows0 = []
+    for row in cur0:
+        rows0.append(row)
+
+    cur1 = presto_connection.cursor()
+    cur1.execute("select custkey from tpch.sf1.customer LIMIT 10")
+    rows1 = cur1.fetchall()
+
+    assert len(rows0) == len(rows1)
```

Comment:
- you need ORDER BY to get stable results
- you can read from `nation`, test will be faster (then no need for LIMIT)

Reply:
Changed to use `nation` and also asserts on the actual rows to in addition to length now.